drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -7957,13 +7957,11 @@ void RenderingDeviceVulkan::compute_list_bind_uniform_set(ComputeListID p_list,
 
 				textures_to_storage[i]->used_in_compute = false;
 				textures_to_storage[i]->used_in_raster = false;
-				textures_to_storage[i]->used_in_compute = false;
```

Comment:
Maybe it's supposed to be `used_in_transfer`?

Reply:
Seems like. Asked in /rendering.

Reply:
Done.